core/backend/src/test/schema/ClassRegistry.test.ts

```diff
@@ -103,13 +103,89 @@ describe("Class Registry", () => {
   });
 });
 
+describe("Class Registry - getRootMetaData", () => {
+  let imodel: SnapshotDb;
+
+  before(async () => {
+    const seedFileName = IModelTestUtils.resolveAssetFile("test.bim");
+    const testFileName = IModelTestUtils.prepareOutputFile("ClassRegistry", "GetRootMetaData.bim");
+    imodel = IModelTestUtils.createSnapshotFromSeed(testFileName, seedFileName);
+    assert.exists(imodel);
+    await imodel.importSchemaStrings([
+      `<?xml version="1.0" encoding="UTF-8"?>
+        <ECSchema schemaName="TestSchema1" alias="ts1" version="01.00" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.1">
```

Comment:
Prefer EC 3.2 schemas when creating new test schemas

Reply:
the profile for the existing seed iModel is too low to use ECXML 3.2, we can upgrade the seed and bump ECXML versions in tests in another PR imo

Reply:
ok, we can skip this for now